cms/djangoapps/contentstore/signals/handlers.py

```diff
@@ -100,7 +102,7 @@ def handle_grading_policy_changed(sender, **kwargs):
         'event_transaction_id': unicode(get_event_transaction_id()),
         'event_transaction_type': unicode(get_event_transaction_type()),
     }
-    result = compute_all_grades_for_course.apply_async(kwargs=kwargs)
+    result = compute_all_grades_for_course.apply_async(kwargs=kwargs, countdown=GRADING_POLICY_COUNTDOWN_SECONDS)
```

Comment:
I'm surprised that we didn't have to update the unit test for `handle_grading_policy_changed` after adding the countdown keyword argument.

Reply:
I'm not sure there is one...